assets/templates/oauth1/complete.html

```diff
@@ -54,6 +55,24 @@
 				margin-right: 4px;
 			}
 		</style>
+		<script>
+			const handleDisconnect = () => {
+				// Splitting the cookies string on the basis of cookie name and then popping out the first value.
+				const token = ('; ' + document.cookie).split('; MMCSRF=').pop().split(';')[0];
+
+				fetch("{{ .RevokeURL }}", {
+					method: "POST",
+					headers: {'X-CSRF-Token': token},
+				}).then(res => {
+					let element = document.getElementsByTagName("body")[0]
+					if(!res.ok) {
+						element.style.margin = "10px"
+      				        }
+
+					return res.text().then(text => element.innerHTML = text)
```

Comment:
```suggestion
					const element = document.body;
					if (!res.ok) {
						element.style.margin = "10px"
					}

					res.text().then(text => element.innerHTML = text);
```

Reply:
Why do we only apply the margin style, when the request fails? Also, can we close this tab if successful?

Reply:
@mickmister 
1. We are applying margin only in case of errors to make the UI look consistent with other error pages.
2. We didn't close the tab because there is a 'Disconnect' button on the result page. If, we close the tab that part will be useless. Let us know your opinions on this.

Reply:
> We didn't close the tab because there is a 'Disconnect' button on the result page

@raghavaggarwal2308 Has the user already disconnected their account at that point? I'm not sure what the value is in showing the button after the POST request is finished.

Reply:
@mickmister My apologies, I got confused with the changes of PR #953. Yes, we can close the tab if successful. I will make that change.

Reply:
Are you consistently using spaces in the editor for the html file? There has to be something weird about the indentation in the file. Any ideas?